survival_guide/1_concepts.adoc

```diff
@@ -23,7 +23,13 @@ This topic covers some of the most important concepts of the Fusion cluster depi
 
 //tag::which[]
 
-Although Figure 1 above depicts running in Google Kubernetes Engine (GKE), Fusion 5 runs on any modern version of Kuberentes (v 1.14 or newer). In fact, very little code in Fusion has any awareness of Kubernetes or Docker. The K8s ecosystem is moving very fast and infrastructure providers are quickly offering some flavor of Kubernetes. However, the beauty of Kubernetes is that applications like Fusion do not need to care about the underlying infrastructure. Although Lucidworks provides setup scripts for GKE, AKS, and EKS to help users get started with Fusion 5, do not take this to mean Fusion only runs on those flavors of Kubernetes. In general, as long as K8s is running, Fusion 5 can run on it.
+Figure 1 above depicts running in Google Kubernetes Engine (GKE). To run Fusion 5, use your cloud provider's stable release channel.
+
+NOTE: GKE refers to the version as *stable release channel*. Other providers may use a different term for the version on which to run Fusion 5. 
+
+Very little code in Fusion has any awareness of Kubernetes or Docker. The K8s ecosystem can change quickly and infrastructure providers offer some flavor of Kubernetes. However, Kubernetes is structured so applications like Fusion do not need to care about the underlying infrastructure. 
```

Comment:
I kind of think we can just cut this paragraph entirely.

Reply:
@gkanapathy do you have an opinion here?

Reply:
Commented out paragraph and made other requested edits. Pushed branch again. Mike or Gerald need decide to either remove the comment // notation to include the paragraph, or delete the paragraph altogether.